packages/untaped-workspace/tests/unit/test_sync_workspaces_use_case.py

```
@@ -0,0 +1,263 @@
+                    "strict_only": strict_only,
+                    "bare_tracker": bare_tracker,
+                }
+            )
+        gate = self._gates.get(workspace.name)
+        if gate is not None:
+            gate.wait(timeout=2.0)
+        return list(self._outcomes.get(workspace.name, []))
+
+
+def _ws(name: str) -> Workspace:
+    return Workspace(name=name, path=Path(f"/tmp/{name}"))
```

Ruff (0.15.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)